File: src/Python/Entrega/adivina_el_numero.py
from random import randint
n = randint(1,20)

print("Bienvenido a 'Adivina el numero'")
print("Estoy pensando en un numero entre 1 y 20")
print("Trata de adivinarlo en no más de 6 intentos")

for i in range (6):
    guess = int(input("Adivina\n"))
    if guess < n:
        print("Es muy bajo")
    elif guess > n:
        print("Es muy alto")
    else:
        break

if guess != n:
    print("Game over, no has podido adivinar el numero en 6 intentos")
    print(f"El numero era {n}")
else:
    print(f"Buen trabajo, has adivinado el numero en {i+1} intentos!")

File: src/_java/unidad0/actividades/AdivinaElNumero.java
package _java.unidad0.actividades;

import java.util.Random;
import java.util.Scanner;

public class AdivinaElNumero {
    public static void main(String[] args) {
        final int MAX_NUMBER = 20;
        final int MAXTRIES = 6;
        Random random = new Random();
        Scanner sc = new Scanner(System.in);
        int randNum = random.nextInt(MAX_NUMBER) + 1;
        int tries = 0;
        int input = 0;

        do {
            System.out.println("Adivina");
            input = sc.nextInt();
            if (input > randNum) {
                System.out.println("Es muy alto");
            } else if (input < randNum) {
                System.out.println("Es muy bajo");
            }
            tries ++;
        } while (input != randNum && tries <= MAXTRIES - 1);
        
        System.out.println("Buen trabajo, has adivinado en " + tries + " intentos!");
        sc.close();
    }
}